PhysicsTools/TensorFlow/test/BuildFile.xml

```diff
@@ -2,38 +2,95 @@
   <use name="tensorflow-cc"/>
   <use name="boost_filesystem"/>
   <use name="cppunit"/>
+  <use name="PhysicsTools/TensorFlow"/>
 </bin>
 
+<iftool name="cuda-gcc-support">
```

Comment:
I _think_ 
```suggestion
<iftool name="cuda">
```
would be more appropriate in this case, because the tests do not depend on compiling any CUDA code, but (indirectly) on the CUDA runtime instead.